### ISSUE TITLE
Donation and Participation page in other languages missing

### ISSUE THREAD

**Issue by Entkenntnis** [serlo/frontend#375]
https://en.serlo.org/mitmachen and https://en.serlo.org/spenden do not exist, the same for all other language instances. Is this a bug or are these pages simply missing?

Also missing: https://en.serlo.org/donate, https://en.serlo.org/jobs. The only page existing is https://en.serlo.org/get-involved

**Comment by inyono** [comment]
Yes, they don't exist. Seems like we currently have the bug in the legacy system, too. Probably should link to de.serlo.org/spenden for now or remove the icons.

**Comment by jakobwes** [comment]
Just saw this issue. The /mitmachen-Seite is currently a normal static page with a different url in every language-version (/get-involved in en.serlo.org, fr.serlo.org and ta.serlo.org and /participa in es.serlo.org). We want the url to be humanly readable, therefore we named the pages like this.
For the jobs and the donate-page: these pages existed as static pages in the english language-version a while back, but I deleted them, because they were outdated and not maintained (and it doesn't at the moment fit in our strategy to maintain/use them) :)
